packages/react-reconciler/src/__tests__/ReactSuspense-test.internal.js

```diff
@@ -241,4 +241,63 @@ describe('ReactSuspense', () => {
     ]);
     expect(root).toMatchRenderedOutput('AsyncSibling');
   });
+
+  // This is a regression test for a Suspense bug.
+  it('specifying larger maxDuration is equivalent to omitting it', () => {
+    [100, 500, 1000, 2000, 5000, 10000, undefined].forEach(maxDuration => {
```

Comment:
I don't know if all of these are a bug. At the very least `[2000, undefined]` seems to demonstrate a bug to me. Others might fail for another (maybe legit?) reason although I don't see it immediately.